docs/sources/mimir/configure/configure-otel-collector.md

```diff
@@ -111,9 +111,13 @@ service:
 
 OpenTelemetry metrics use resource attributes to describe the set of characteristics associated with a given resource, or entity, producing telemetry data. For example, a host resource might have multiple attributes, including an ID, an image, and a type.
 
-To optimize the storage of and ability to query this data, you can configure Mimir to promote specified OTel resource attributes to labels at the time of ingestion, with periods (`.`) replaced by underscores (`_`).
+To optimize the storage of and ability to query this data, you can use the `promote_otel_resource_attributes` setting to configure Mimir to promote specified OTel resource attributes to labels at the time of ingestion.
```

Comment:
Nits: From looking at something like the [shuffle sharding doc](https://github.com/grafana/mimir/blob/2e2dccddb3bd8245a088ca44cb48feb2f48dc786/docs/sources/mimir/configure/configure-shuffle-sharding/index.md) or [configure custom trackers doc](https://github.com/grafana/mimir/blob/2e2dccddb3bd8245a088ca44cb48feb2f48dc786/docs/sources/mimir/configure/configure-custom-trackers.md) we usually refer to these as "options" rather than "settings", and it seems like we try to use the CLI flag form `-distributor.otel-promote-resource-attributes` in docs rather than the YAML form.

Reply:
Good call out 😄 Updated!